### ISSUE TITLE
Gegenentwurf Musteranschreiben

### ISSUE THREAD

**Issue by TaherP** [studicouch/1Tag1Klinik#12]
**Gegenentwurf**

Sehr geehrter Herr Prof. ....

Wir brauchen Ihre Unterstützung! Ich bin Mitgründer des studentischen Projekts "1Klinik1Tag" und wir wollen mit Ihnen Studierenden ein innovatives Lehrkonzept in Form von 1-Tages-Hospitationen anbieten. Sie haben die Möglichkeit, Studierende mit Interesse und Eigeninitiative für Ihr Fach kennenzulernen und ihnen einen Einblick in Ihr Fachgebiet zu zeigen. Der Tag soll eine Inspiration sein und eine Perspektive für folgende Famulaturen, das PJ und die Assistenzzeit bieten. 

Sie ermöglichen Studierenden, in Kontakt mit einem Fachgebiet zu bekommen, das sie schon immer interessiert hat. Eine gute Betreuung und genügend Zeit für Fragen heben dieses Angebot vom Uniangebot hervor. Lassen Sie Ihre Motivation und Leidenschaft für den Beruf blühen und überzeugen Sie mehr Nachwuchsärzte für Ihr Fach und Ihre Klinik.

Wenn wir Ihr Interesse geweckt haben, zögern Sie nicht, mich per Mail oder telefonisch unter ... zu kontaktieren. 

Wir haben schon zahlreiche renommierte Kliniken als Kooperationspartner gewonnen. Machen Sie sich gernen einen ersten Eindruck unter www.1Klinik1Tag.de.

Mit freundlichen Grüßen

Cand.med. Glas

Studierende Pros:
Fach, das sie interessiert
Gute Betreuung 
vielfältige und realistische Einblicke
Raum für Fragen zu dem Beruf
Zukunftsperspektive sehen

Klinik Pros:
Klinik repräsentieren
Werbung für das Fach machen
Attraktivität für Praktika, Doktorarbeiten, Arztstellen
Teil eines neuen Lehrkonzepts werden
Deutsche Kliniken attraktiv bleiben lassen

Anschreiben Klinik Hancken:
Sehr geehrte Frau Feil,

vielen Dank für das nette Telefonat am 22.02.2016.

Wir brauchen Ihre Unterstützung! Ich bin Mitgründer des studentischen Projekts "1Klinik1Tag" und wir wollen mit Ihnen Studierenden ein innovatives Lehrkonzept in Form von 1-Tages-Hospitationen anbieten. Sie haben die Möglichkeit, Studierende mit Interesse für Radiologie kennenzulernen und ihnen einen eindrücklichen Einblick zu ermöglichen. Der Tag soll eine Inspiration sein und eine Perspektive bieten für Famulaturen, das PJ und die Assistenzzeit.

Das Konzept gibt es schon seit wenigen Jahren bei der Deutschen Gesellschaft für Neurologie http://www.junge-neurologen.de/aktivitaeten/dein-tag-in-der-neurologie. Gerne möchten wir dieses Angebot für Kliniken anbieten, mit denen Studierende normalerweise nicht in Kontakt kommen.

Eine gute Betreuung und genügend Zeit für Fragen heben dieses Angebot vom täglichen Uniangebot hervor. Lassen Sie Ihre Motivation und Leidenschaft für den Beruf blühen und überzeugen Sie mehr Nachwuchsärzte für die Radiologie und die Klinik Hancken.

Wenn wir Ihr Interesse geweckt haben, zögern Sie nicht, mich per Mail oder telefonisch unter 01732667115 zu kontaktieren. Ich kenne die Klinik aus meinem PJ in Stade und habe durchweg positives erfahren. Als Arzt möchte ich gerne den Studierenden ermöglichen, was ich mir selbst gewünscht habe.

Mit freundlichen Grüßen

Taher Pham

**Comment by Frederik90** [comment]
Finde ich in weiten Teilen sehr gut! Ich denke, dass diejenigen unter uns, die ihre Kontakte anschreiben, ohnehin ein individualisiertes Anschreiben verwenden und sich bei Bedarf aus meinem und Tahers Vorschlag dass passende zusammen kopieren können. 

Perspektivisch bietet es sich sicher an verschiedene Musterschreiben zu haben, beispielsweise in den Kategorien:
- förmlich
- locker-studentisch
- persönlich 
  (ich weiß, auf den ersten Blick widerspricht sich "persönliches Musteranschreiben" etwas, aber einige Dinge lassen sich sicher auch bei "persönlichen" Anschreiben wiederverwenden.)

**Comment by TaherP** [comment]
Ja, finde ich sinnvoll. Wir sollten dann beide Anschreiben trimmen und dann können sie auch schon losgeschickt werden. Ich schick mir meine Verbesserungen am WE

**Comment by AliMalikAl** [comment]
Kleine Frage vorweg: Sollten wir nicht vllt im Einleitungssatz schreiben "ich bin MITgründer" statt "ich bin Gründer" ? Ich finde das kommt etwas besser rüber,oder was denkt ihr?

**Comment by TaherP** [comment]
Ja, finde ich sinnvoll. Wurde übernommen!

**Comment by AliMalikAl** [comment]
Sollen wir unsere Anschreiben dann auch hier auf Github dokumentieren?

**Comment by TaherP** [comment]
Ja auf jeden Fall. Für Feedback und Verbesserung. Jeder bringt andere Punkte ein und irgendwann machen wir eine allgemeine Formulierung, da fließt dann alles rein
